packages/ripple/README.md

```diff
@@ -0,0 +1,43 @@
+# Ripple
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/ripple.svg)
+
+#### [View on Storybook](https://mongodb.github.io/leafygreen-ui/?path=/story/ripple--default)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/ripple
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/ripple
+```
+
+## Example
```

Comment:
Probably should show vanilla JS example too!

Reply:
Makes sense, will add.